var/spack/repos/builtin/packages/py-pygelf/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPygelf(PythonPackage):
+    """Python logging handlers with GELF (Graylog Extended Log Format) support."""
+
+    homepage = "https://github.com/keeprocking/pygelf"
+    url      = "https://files.pythonhosted.org/packages/5e/4f/a6224902db19be061ffb090a247014c4716d45e5c97ae4d143321e818092/pygelf-0.3.6.tar.gz"
```

Comment:
```suggestion
    url      = "https://pypi.io/packages/source/p/pygelf/pygelf-0.3.6.tar.gz"
```
This allows us to fetch multiple versions with a single URL